themes/digitalservices/layouts/partials/header.html

```diff
@@ -60,6 +60,8 @@
         class="jobs"
     {{ end }}
 >
+    <a href="#main" class="element-invisible element-focusable" id="skip-link" tabindex="0">Skip to main content</a>
```

Comment:
I think maybe it doesn't work anymore? if I hit enter, then keep tabbing it stays on the top row

Reply:
try now?